src/overlays/actors/ovl_Eff_Zoraband/z_eff_zoraband.c

```diff
@@ -30,16 +30,71 @@ const ActorInit Eff_Zoraband_InitVars = {
     (ActorFunc)EffZoraband_Draw,
 };
 
-#endif
+void EffZoraband_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EffZoraband* this = THIS;
 
-extern UNK_TYPE D_06000F38;
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = EffZoraband_MikauFadeOut;
+    this->alpha = 0;
+    this->actor.shape.rot.z = 0;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Zoraband/EffZoraband_Init.s")
+void EffZoraband_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Zoraband/EffZoraband_Destroy.s")
+void EffZoraband_MikauFadeOut(EffZoraband* this, GlobalContext* globalCtx) {
+    if (Cutscene_CheckActorAction(globalCtx, 0x20F) != 0) {
```

Comment:
```suggestion
    if (Cutscene_CheckActorAction(globalCtx, 0x20F)) {
```
Looking at the function, it returns a boolean.